openpgp/add.go

```diff
@@ -405,9 +371,9 @@ UPDATE openpgp_pubkey SET revsig_uuid = $1 WHERE uuid = $2`,
 
 func (w *Worker) updateSubkeyRevsig(tx *sqlx.Tx, subkey *Subkey, r *Signature) error {
```

Comment:
There's already an `updater` available where these methods are called from -- I think it makes more sense to pass in the updater than the transaction.